File: 414/414.py
from sys import stdin
from functools import reduce

nImages = int(input(''))

while nImages != 0 :
    results = []
    totalSpaces = 0
    for n in range(0, nImages):
        cont = 0
        line = input('')
        
        for i in range(0, 25):            
            if line[i].isspace():
                cont += 1
        results.append(cont)
    
    minVal = min(results)
    for nSpaces in results:
        totalSpaces += nSpaces - minVal
    print(totalSpaces)
    nImages = int(input('')) 

File: 414/414.cpp
#include <bits/stdc++.h>
using namespace std;

main() {
    int nCases, minSpaces, totalSpaces;
    string line;
    
    while(cin >> nCases && nCases != 0){
        cin.ignore();
        vector<int> nSpaces(nCases);
        minSpaces = 100;
        totalSpaces = 0;
        for(int i=0; i<nCases; i++){
            int cont = 0;
            getline(cin, line);
            for(int j=0; j<25; j++) {
                if(line[j] == ' ')
                    cont++;
            }
            nSpaces[i] = cont;
            if(nSpaces[i] < minSpaces)
                minSpaces = nSpaces[i];
        }
        for(int i=0; i<nCases; i++) {
            totalSpaces += nSpaces[i] - minSpaces;
        }
        cout << totalSpaces << endl;
    }       
}